### ISSUE TITLE
Test addresses should be added based on chaintype

### ISSUE THREAD

**Issue by gilescope** [polkadot-js/apps#6752]
At the moment test addresses like Alice, Bob etc. are added to the addresses page if the chainspec's name is "Development". Basti says they should really only be added if the chaintype is `ChainType::Development`. The ChainType has been added to improve on us having to check for certain names.

(See: https://stackoverflow.com/questions/70508561/where-are-accounts-located-in-substrate-node-template/70519554#70519554 )

I looked but couldn't quite find where this is happening in the page-accounts. Happy to do a PR if someone can point me at the right bit.

**Comment by jacogr** [comment]
It checks for 3 things -

- ChainType.isDevelopment 
- ChainType.IsLocal
- finally, on the name

https://github.com/polkadot-js/apps/blob/master/packages/react-api/src/Api.tsx#L135

TL;DR SO is wrong, or rather pre-dates the introduction of the `system_chainType` RPC (the name check is done for historic reasons only, hence the other 2 being first)

**Comment by gilescope** [comment]
Ah it's in the common repo. Ok I see it's the fallback that's triggering it. Will update the SO post.
